### ISSUE TITLE
National notice was cancelled but the cancelledReason text is missing in Hilma

### ISSUE THREAD

**Issue by esaiko** [Hankintailmoitukset/hilma-api#72]
**Describe the bug**
User cancelled a National notice (type=9902).
Notice's json has properties isCancelled and cancelledReason as in the specs.
Hilma accepted the notice and the status was changed to cancelled and this can be seen in Hilma.
However, the text passed in cancelledReason is NOT visible in Hilma.

**To Reproduce**
````
PUT https://api.hankintailmoitukset.fi/ets/external-api/v1/notice/999?projectId=888&parentId=777
....
{
    "type": 9902,
...
"isCancelled": true,
    "cancelledReason": [
        "Tarjouskilpailu keskeytet\u00e4\u00e4n, koska kustannukset nousivat selke\u00e4sti yli budjettiin varattujen
 varojen. Kts.liite"
    ]
}
````
View the notice in Hilma: https://www.hankintailmoitukset.fi/fi/public/procurement/39793/notice/54980/overview
The status is displayed as "cancelled" (which is correct) but the cancelledReason text is not displayed. 
GET https://api.hankintailmoitukset.fi/web/api/notices/54980 returns json with empty cancelledReason.

**Expected behavior**
The cancelledReason text should be stored by Hilma and displayed properly.
**API response**
````
PUT https://api.hankintailmoitukset.fi/ets/external-api/v1/notice/99999?projectId=99999&parentId=99999
Content-Length: 6322
Ocp-Apim-Subscription-Key: ed4xxx
Content-Type: application/json

{
    "type": 9902,
...
"isCancelled": true,
    "cancelledReason": [
        "Tarjouskilpailu keskeytet\u00e4\u00e4n, koska kustannukset nousivat selke\u00e4sti yli budjettiin varattujen varojen. Kts.liite"
    ]
}

201 Created
Cache-Control: no-store,no-cache
Pragma: no-cache
Transfer-Encoding: chunked
Content-Type: application/json; charset=utf-8
Location: http://hns-hilma-prod-api-wapp.azurewebsites.net/external-api/v1/notice/9999
Request-Context: appId=cid-v1:1c7c1805-f4ec-49e2-a8f3-5993b60c63f5
X-Azure-Ref: 0fSp0XwAAAABO66GShoU8T5lfU9rAMgHLQlJVMzBFREdFMDQxMwAwMTEzMDRiZi1jMzE2LTQ1YTAtYWVhOC0xYWQwMGMyZTZiMjc=
Date: Wed, 30 Sep 2020 06:49:32 GMT
Connection: close


{"noticeNumber":"2020-054980","creationDate":"2020-09-30T06:49:33.1175659Z","modificationDate":"2020-09-30T06:49:33.2171858Z","hilmaStatus":2,"hilmaPublicationDate":"2020-09-30T06:49:33.2121858Z","etsIdentifier":"88888","organisationId":"0129261-5","projectId":39793,"noticeId":54980,"tedStatus":0}
````

**Comment by Zamm** [comment]
A fix for this  problem  is coming soon. This notice (54980) is fixed now. we added the missing cancelation reason to the notice.

**Comment by Zamm** [comment]
A fix is currently installed to the staging environment.
Can you validate the fix by canceling a national notice in the staging environment.

**Comment by jannef** [comment]
A fix for this issue was installed 23.11.2020, around 15:30
